### PR TITLE
Add check for ocamlopt since it is used

### PR DESCRIPTION
**Pull request by aserdean**
Found by inspection.

Signed-off-by: Alin Gabriel Serdean <aserdean@ovn.org>

### CODE REVIEW

File: tools/oeedger8r/CMakeLists.txt [hunk 1]
```diff
@@ -31,6 +31,10 @@ add_custom_command(
   DEPENDS intel/Parser.mly)
 
 # Compile
+find_program(OCAMLOPT ocamlopt)
```
Comment (andschwa) [inline tools/oeedger8r/CMakeLists.txt]: Should we take this and the checks for ocamllex and ocamlyacc and do them in one place? Then we can do one if (NOT OR NOT OR NOT) with a single error message. Up to you.

Comment (aserdean) [reply tools/oeedger8r/CMakeLists.txt]: Yup that would look cleaner. I updated the commit.

**Comment by andschwa** [comment]
bors r+

@johnkord Did we already / are we working to temporarily disable the flaky tests?

**Pull request merged**